exercises/practice/paasio/paasio.go

```diff
@@ -1 +1,33 @@
 package paasio
+
+import "io"
+
+// Define readCounter and writeCounter types here.
+// For the return of the function NewReadWriteCounter, you must also define a type that satisfies the ReadWriteCounter interface.
+func NewWriteCounter(writer io.Writer) WriteCounter {
```

Comment:
```suggestion
// Define readCounter and writeCounter types here.

// For the return of the function NewReadWriteCounter, you must also define a type that satisfies the ReadWriteCounter interface.

func NewWriteCounter(writer io.Writer) WriteCounter {
```

Just added more spaces to make it clear that they are separate things and they are not related with the `NewWriteCounter` function.